code-dot-org/code-dot-org
# show error details when creating level via AddLevelDialog fails

Follow up from https://github.com/code-dot-org/code-dot-org/pull/43095

## Testing story

manually verified as follows:

level name already taken

![Screen Shot 2021-10-21 at 12 30 39 PM](https://user-images.githubusercontent.com/8001765/138344456-816cd478-fdc4-4621-85b3-5317cbb836b9.png)

known issue for dsl levels, see previous PR for notes 

![Screen Shot 2021-10-21 at 12 24 52 PM](https://user-images.githubusercontent.com/8001765/138344320-33014f3e-9126-458e-8fd0-22ed9eeab9e2.png)

once dsl levels are fixed, we should be able to see the full error message from https://github.com/code-dot-org/code-dot-org/pull/43095 too.
